fr.free.brouits.qspeakers.yml

```diff
@@ -0,0 +1,23 @@
+app-id: fr.free.brouits.qspeakers
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: qspeakers
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  - --device=dri
+  # Needs to save files locally
+  - --filesystem=xdg-documents
```

Comment:
Are you sure this is needed? It seems to use filechooser portal on my end.

Reply:
I am not sure it is needed. The app uses the filechooser to save documents, but maybe it is a delegation that doesn't require permission `--filesystem=xdg-documents` ? Another place is used too for the app, `~/.local/share/Herewe/QSpeakers/` to store db and current project. Does it need a permission ?

Reply:
As long as it uses the native file chooser, it doesn't need static filesystem permissions here. Portals will grant and expose chosen paths for the app.

> ~/.local/share/Herewe/QSpeakers/
As long as your app respects $XDG_CONFIG_HOME and $XDG_DATA_HOME, access to these paths is also granted automatically.